UI/src/model/selectedSpeedLimit.js

```diff
@@ -3,13 +3,13 @@
     var selection = [];
     var self = this;
     var dirty = false;
-    var originalSpeedLimitValue = null;
+    var originalSpeedLimitValues = null;
     var isSeparated = false;
 
     this.splitSpeedLimit = function(id, split) {
       collection.splitSpeedLimit(id, split, function(splitSpeedLimits) {
         selection = [splitSpeedLimits.created, splitSpeedLimits.existing];
-        originalSpeedLimitValue = splitSpeedLimits.existing.value;
+        originalSpeedLimitValues = { isSuggested: splitSpeedLimits.existing.value.isSuggested, value: splitSpeedLimits.existing.value.value };
```

Comment:
Can this "value.value" be changed to something more explicit.

Reply:
Let's leave that "value.value" like it is because to switch that we will have a lot of impact on the rest of the code